app/views/admin/statistics_announcements/_filter_options.html.erb

```diff
@@ -0,0 +1,80 @@
+<%
+  anchor ||= anchor || ""
+  raise "filter action required" unless defined?(filter_action)
```

Comment:
I think we can just directly pop our filter action into the form itself here as this isn't a shared partial (the editions filter is used in 3 different places)

So we can just do 

```
 form_with url: admin_statistics_announcements_path + anchor, method: :get do |_form| %>

```

& 

```
    <p class="govuk-body"><%= link_to "Reset all fields",  admin_statistics_announcements_path + "?state=active" + anchor, class: "govuk-link" %></p>

```

Reply:
Nice, yes, much tidier. And we don't need the anchor.